lib/token-autocomplete.ts

```diff
@@ -1092,6 +1093,7 @@ class TokenAutocomplete {
          * Shows the suggestions dropdown to the user.
          */
         showSuggestions() {
+            this.container.classList.add('select-arrow-active');
```

Comment:
**Optional:** I would prefer if the class name would also be prefixed with `token-autocomplete-`